### DISCUSSION TITLE
Sending a array with objects

### DISCUSSION THREAD

**Discussion by bytesagar** [prisma/prisma#10463] [Q&A] [answered]
Can we send array with objects to prisma. 

```
[
{
     name:"someone",
     age:"12",
},
{
     name:"someone",
     age:"12",
},
{
     name:"someone",
     age:"12",
},
{
     name:"someone",
     age:"12",
},
{
     name:"someone",
     age:"12",
},


]
```

I want to send this array to my database. I also checked scalar and json type but didnt get it.

**Comment by TasinIshmam** [comment]
Hey @bytesagar 

Could you please provide your current schema? That way, it might be easier to give a concrete suggestion. 

Generally speaking, you can pass an array of objects to `createMany` to create multiple records.

**Reply by bytesagar** [reply]
@TasinIshmam 

My complients model
```
model Complient {
  id        Int       @id @default(autoincrement())
  createdAt DateTime  @default(now())
  updatedAt DateTime? @updatedAt
  name      String
  patientId String    @unique
  patient   Patient?  @relation(fields: [patientId], references: [patient_id])
}
```

My model user
```
model Patient {
 id        Int       @id @default(autoincrement())
patient_id String
 chief_complients Complient[]

}
```

I want to send that entire array at once and update the User. How can I achieve it

**Reply by TasinIshmam** [reply]
@bytesagar  Could you clarify which table you are trying to update? There doesn't seem to be a table with a `name` and `age` property. 

Additionally, I could you clarify if you want to `create` new records or `update` existing records?

**Reply by bytesagar** [reply]
@TasinIshmam it was just a dummy data
This is the data i want to create on user table of `Complients` .
```
[
   {name: "complient 1"},
   {name: "complient2"},
   {name: "complient3"}
]
```

I want to create Complient with this array of objects. So my query was how can i send the array

**Reply by TasinIshmam** [reply, marked as answer]
Hi @bytesagar 

Sorry for the delay. You can pass that array to `createMany` to create the `complient` records. Are these records associated with a certain `Patient` record? In that case, each object should also have a `patientId` field mentioning the foreign key. Here is an [example](https://www.prisma.io/docs/concepts/components/prisma-client/crud#create-multiple-records) of how to pass an array to `createMany`.

Let me know if this clear :smile:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
